### DISCUSSION TITLE
Solve break code or pin griffe because recently has updated?

### DISCUSSION THREAD

**Discussion by Anavelyz** [machow/quartodoc#356] [General]
Recently, `griffe` had been updated, and I think it is cause to get the following error: 

`ModuleNotFoundError: No module named 'griffe.loader'`

I restrict the version of `griffe` in my project and it works fine.

**Comment by machow** [comment]
Sorry for the slow reply, but in case others come to this discussion, quartodoc v0.7.6 should work with the recent griffe v1+ release (and earlier versions)
